packages/dayjs/src/dayjs-utils.ts

```diff
@@ -273,6 +273,14 @@ export default class DayjsUtils<TDate extends Dayjs = Dayjs> implements IUtils<T
     return ampm === "am" ? "AM" : "PM";
   };
 
+  public startOfYear = (date: Dayjs) => {
+    return date.clone().startOf("year") as TDate;
```

Comment:
I think you are doing double clone here https://day.js.org/docs/en/manipulate/start-of because start of cloning the value itself.

Reply:
```ts
  public startOfMonth = (date: Dayjs) => {
    return date.clone().startOf("month") as TDate;
  };
```

We also clone for `startOfMonth` and `startOfWeek`

`endOf` / `add` also clone.
I will do a follow up PR to remove the useless clones :+1: 

Done for `startOfYear` / `endOfYear`